Dockerfile

```diff
@@ -35,12 +35,17 @@ RUN groupadd -g 1000 paperless \
     && useradd -u 1000 -g 1000 -d /usr/src/paperless paperless \
     && chown -Rh paperless:paperless /usr/src/paperless
 
+# Set export directory
+ENV PAPERLESS_EXPORT_DIR /export
+RUN mkdir -p $PAPERLESS_EXPORT_DIR
+RUN chown -Rh paperless:paperless $PAPERLESS_EXPORT_DIR
```

Comment:
Running this line during the build will not reliably work if the end-user host-mounts the export directory, this needs to be done in the `docker-entrypoint.sh` script. 

You can pretty much duplicate/copy-paste the logic from the [consumption directory](https://github.com/danielquinn/paperless/blob/819a0e1f570aa5e65d2bf3aa1b98e0bc13c19196/scripts/docker-entrypoint.sh#L18-L36).